oml/metrics/embeddings.py

```diff
@@ -188,9 +195,12 @@ def compute_metrics(self) -> TMetricsDict_ByLabels:  # type: ignore
             mask_to_ignore=self.mask_to_ignore,
             check_dataset_validity=self.check_dataset_validity,
             reduce=False,
-            **args,  # type: ignore
+            **args_retrieval_metrics,  # type: ignore
         )
 
+        embeddings = self.acc.storage[self.embeddings_key]
+        metrics[self.overall_categories_key].update(calc_topological_metrics(embeddings, **args_topological_metrics))
```

Comment:
let's replace update with `=` by the analogue of retrieval metrics

Reply:
Do you mean we need to add additional keys to distinguish the retrieval metrics and topological? Like the following

```python
metrics[self.overall_categories_key]['retrieval'] = calc_retrieval_metrics(...)
metrics[self.overall_categories_key]['topological'] = calc_topological_metrics(...)
```

Reply:
please, ignore this comment, I misunderstood what was going on there